<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugForm</class>
 <widget class="QWidget" name="debugForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>debugForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>90</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>当前窗口标题：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="titletext">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>6</y>
     <width>190</width>
     <height>49</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
